app/posts/apply-for-teacher-training/2023-10-31-blocking-candidates-with-no-visa.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Blocking candidates from applying for courses that do not sponsor visas
+description:
+date: 2023-10-31
+tags:
+  - visa sponsorship
+  - international
+---
+
+There was an agreement that it is beneficial to block certain user groups from submitting an application. Specifically when a course will not sponsor visas, and the candidate does not have the right to work or study in the UK.
+
+## The issue
+
+In 2022-2023 cycle we have had over 4,800 candidate applications rejected due to not having visa sponsorship. This wastes candidate and provider time, whilst also reducing candidates' chance of finding a course match.
+
+In 2023-2024 cycle there was a high proportion of rejection on day one due to no visa.
+
+We used to have a blue inset text on the application tab, once they have already applied to the course, warning them that this course does not offer visa sponsorship and that they need to delete their application and to look for a different course that does offer visa sponsorship. We have found that it doesnt stop the candidates sufficiently, and needed a different way to stop them.
+
+![Screenshot of how we used to tell candidates that the course does not sponsor visas](before.png)
+
+## What we did
+
+We have revisited the user journey from when they fill out their details in 'Your details' tab to adding an application on 'Your application' tab, to include the logic that when a candidate answers the question about the 'right to work or study in UK' with 'No' in their 'personal information' section, to not allow them to add an application that does not have visa sponsorship, and they would be displayed with a warning and link to Find service.
+
+![Screenshot of how the application review page looks like with the blocking pattern](after.png)
+
+We went with the above screenshot example, because:
```

Comment:
```suggestion
In the content for the warning message, we added a link to Find postgraduate teacher training (Find) so that candidates could easily find a different and more suitable course.
```